### PR TITLE
Get rid of generate_tests from html/

### PR DESCRIPTION
**Pull request by gsnedders**

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260680356)

# Firefox (nightly) #
Testing web-platform-tests at revision 73e7c426f7eb7b7c557fb1ef36e9734724a72b63
Using binary /home/travis/build/w3c/web-platform-tests/_venv/firefox/firefox
Starting tests
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/6738/html/semantics/scripting-1/the-template-element/template-element/template-content.html">/html/semantics/scripting-1/the-template-element/template-element/template-content.html</a></summary>


|                                                  Subtest                                                  | Results |                    Messages                    |
|-----------------------------------------------------------------------------------------------------------|---------|------------------------------------------------|
|                                                                                                           | OK      |                                                |
| `Template may contain a element`                                                                          | PASS    |                                                |
| `Template may contain abbr element`                                                                       | PASS    |                                                |
| `Template may contain address element`                                                                    | PASS    |                                                |
| `Template may contain area element`                                                                       | PASS    |                                                |
| `Template may contain article element`                                                                    | PASS    |                                                |
| `Template may contain aside element`                                                                      | PASS    |                                                |
| `Template may contain audio element`                                                                      | PASS    |                                                |
| `Template may contain b element`                                                                          | PASS    |                                                |
| `Template may contain base element`                                                                       | PASS    |                                                |
| `Template may contain bdi element`                                                                        | PASS    |                                                |
| `Template may contain bdo element`                                                                        | PASS    |                                                |
| `Template may contain blockquote element`                                                                 | PASS    |                                                |
| `Template may contain br element`                                                                         | PASS    |                                                |
| `Template may contain button element`                                                                     | PASS    |                                                |
| `Template may contain canvas element`                                                                     | PASS    |                                                |
| `Template may contain caption element`                                                                    | PASS    |                                                |
| `Template may contain cite element`                                                                       | PASS    |                                                |
| `Template may contain code element`                                                                       | PASS    |                                                |
| `Template may contain col element`                                                                        | PASS    |                                                |
| `Template may contain colgroup element`                                                                   | PASS    |                                                |
| `Template may contain command element`                                                                    | PASS    |                                                |
| `Template may contain datalist element`                                                                   | PASS    |                                                |
| `Template may contain dd element`                                                                         | PASS    |                                                |
| `Template may contain del element`                                                                        | PASS    |                                                |
| `Template may contain details element`                                                                    | PASS    |                                                |
| `Template may contain dfn element`                                                                        | PASS    |                                                |
| `Template may contain dialog element`                                                                     | PASS    |                                                |
| `Template may contain div element`                                                                        | PASS    |                                                |
| `Template may contain dl element`                                                                         | PASS    |                                                |
| `Template may contain dt element`                                                                         | PASS    |                                                |
| `Template may contain em element`                                                                         | PASS    |                                                |
| `Template may contain embed element`                                                                      | PASS    |                                                |
| `Template may contain fieldset element`                                                                   | PASS    |                                                |
| `Template may contain figcaption element`                                                                 | PASS    |                                                |
| `Template may contain figure element`                                                                     | PASS    |                                                |
| `Template may contain footer element`                                                                     | PASS    |                                                |
| `Template may contain form element`                                                                       | PASS    |                                                |
| `Template may contain h1 element`                                                                         | PASS    |                                                |
| `Template may contain h2 element`                                                                         | PASS    |                                                |
| `Template may contain h3 element`                                                                         | PASS    |                                                |
| `Template may contain h4 element`                                                                         | PASS    |                                                |
| `Template may contain h5 element`                                                                         | PASS    |                                                |
| `Template may contain h6 element`                                                                         | PASS    |                                                |
| `Template may contain header element`                                                                     | PASS    |                                                |
| `Template may contain hgroup element`                                                                     | PASS    |                                                |
| `Template may contain hr element`                                                                         | PASS    |                                                |
| `Template may contain i element`                                                                          | PASS    |                                                |
| `Template may contain iframe element`                                                                     | PASS    |                                                |
| `Template may contain img element`                                                                        | PASS    |                                                |
| `Template may contain input element`                                                                      | PASS    |                                                |
| `Template may contain ins element`                                                                        | PASS    |                                                |
| `Template may contain kbd element`                                                                        | PASS    |                                                |
| `Template may contain keygen element`                                                                     | PASS    |                                                |
| `Template may contain label element`                                                                      | PASS    |                                                |
| `Template may contain legend element`                                                                     | PASS    |                                                |
| `Template may contain li element`                                                                         | PASS    |                                                |
| `Template may contain link element`                                                                       | PASS    |                                                |
| `Template may contain map element`                                                                        | PASS    |                                                |
| `Template may contain mark element`                                                                       | PASS    |                                                |
| `Template may contain menu element`                                                                       | PASS    |                                                |
| `Template may contain meta element`                                                                       | PASS    |                                                |
| `Template may contain meter element`                                                                      | PASS    |                                                |
| `Template may contain nav element`                                                                        | PASS    |                                                |
| `Template may contain noscript element`                                                                   | PASS    |                                                |
| `Template may contain object element`                                                                     | PASS    |                                                |
| `Template may contain ol element`                                                                         | PASS    |                                                |
| `Template may contain optgroup element`                                                                   | PASS    |                                                |
| `Template may contain option element`                                                                     | PASS    |                                                |
| `Template may contain output element`                                                                     | PASS    |                                                |
| `Template may contain p element`                                                                          | PASS    |                                                |
| `Template may contain param element`                                                                      | PASS    |                                                |
| `Template may contain pre element`                                                                        | PASS    |                                                |
| `Template may contain progress element`                                                                   | PASS    |                                                |
| `Template may contain q element`                                                                          | PASS    |                                                |
| `Template may contain rp element`                                                                         | PASS    |                                                |
| `Template may contain rt element`                                                                         | PASS    |                                                |
| `Template may contain ruby element`                                                                       | PASS    |                                                |
| `Template may contain s element`                                                                          | PASS    |                                                |
| `Template may contain samp element`                                                                       | PASS    |                                                |
| `Template may contain script element`                                                                     | PASS    |                                                |
| `Template may contain section element`                                                                    | PASS    |                                                |
| `Template may contain select element`                                                                     | PASS    |                                                |
| `Template may contain small element`                                                                      | PASS    |                                                |
| `Template may contain source element`                                                                     | PASS    |                                                |
| `Template may contain span element`                                                                       | PASS    |                                                |
| `Template may contain strong element`                                                                     | PASS    |                                                |
| `Template may contain style element`                                                                      | PASS    |                                                |
| `Template may contain sub element`                                                                        | PASS    |                                                |
| `Template may contain table element`                                                                      | PASS    |                                                |
| `Template may contain tbody element`                                                                      | PASS    |                                                |
| `Template may contain td element`                                                                         | PASS    |                                                |
| `Template may contain textarea element`                                                                   | PASS    |                                                |
| `Template may contain tfoot element`                                                                      | PASS    |                                                |
| `Template may contain th element`                                                                         | PASS    |                                                |
| `Template may contain thead element`                                                                      | PASS    |                                                |
| `Template may contain time element`                                                                       | PASS    |                                                |
| `Template may contain title element`                                                                      | PASS    |                                                |
| `Template may contain tr element`                                                                         | PASS    |                                                |
| `Template may contain track element`                                                                      | PASS    |                                                |
| `Template may contain u element`                                                                          | PASS    |                                                |
| `Template may contain ul element`                                                                         | PASS    |                                                |
| `Template may contain var element`                                                                        | PASS    |                                                |
| `Template may contain video element`                                                                      | PASS    |                                                |
| `Template may contain wbr element`                                                                        | PASS    |                                                |
| `Template may contain a element. The template element and contents are added via body.innerHTML`          | PASS    |                                                |
| `Template may contain abbr element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain address element. The template element and contents are added via body.innerHTML`    | PASS    |                                                |
| `Template may contain area element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain article element. The template element and contents are added via body.innerHTML`    | PASS    |                                                |
| `Template may contain aside element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain audio element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain b element. The template element and contents are added via body.innerHTML`          | PASS    |                                                |
| `Template may contain base element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain bdi element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain bdo element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain blockquote element. The template element and contents are added via body.innerHTML` | PASS    |                                                |
| `Template may contain br element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain button element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain canvas element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain caption element. The template element and contents are added via body.innerHTML`    | PASS    |                                                |
| `Template may contain cite element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain code element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain col element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain colgroup element. The template element and contents are added via body.innerHTML`   | PASS    |                                                |
| `Template may contain command element. The template element and contents are added via body.innerHTML`    | PASS    |                                                |
| `Template may contain datalist element. The template element and contents are added via body.innerHTML`   | PASS    |                                                |
| `Template may contain dd element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain del element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain details element. The template element and contents are added via body.innerHTML`    | PASS    |                                                |
| `Template may contain dfn element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain dialog element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain div element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain dl element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain dt element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain em element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain embed element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain fieldset element. The template element and contents are added via body.innerHTML`   | PASS    |                                                |
| `Template may contain figcaption element. The template element and contents are added via body.innerHTML` | PASS    |                                                |
| `Template may contain figure element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain footer element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain form element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain h1 element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain h2 element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain h3 element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain h4 element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain h5 element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain h6 element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain header element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain hgroup element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain hr element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain i element. The template element and contents are added via body.innerHTML`          | PASS    |                                                |
| `Template may contain iframe element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain img element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain input element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain ins element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain kbd element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain keygen element. The template element and contents are added via body.innerHTML`     | FAIL    | `assert_not_equals: got disallowed value null` |
| `Template may contain label element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain legend element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain li element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain link element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain map element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain mark element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain menu element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain meta element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain meter element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain nav element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain noscript element. The template element and contents are added via body.innerHTML`   | PASS    |                                                |
| `Template may contain object element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain ol element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain optgroup element. The template element and contents are added via body.innerHTML`   | PASS    |                                                |
| `Template may contain option element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain output element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain p element. The template element and contents are added via body.innerHTML`          | PASS    |                                                |
| `Template may contain param element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain pre element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain progress element. The template element and contents are added via body.innerHTML`   | PASS    |                                                |
| `Template may contain q element. The template element and contents are added via body.innerHTML`          | PASS    |                                                |
| `Template may contain rp element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain rt element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain ruby element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain s element. The template element and contents are added via body.innerHTML`          | PASS    |                                                |
| `Template may contain samp element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain script element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain section element. The template element and contents are added via body.innerHTML`    | PASS    |                                                |
| `Template may contain select element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain small element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain source element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain span element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain strong element. The template element and contents are added via body.innerHTML`     | PASS    |                                                |
| `Template may contain style element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain sub element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain table element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain tbody element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain td element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain textarea element. The template element and contents are added via body.innerHTML`   | PASS    |                                                |
| `Template may contain tfoot element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain th element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain thead element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain time element. The template element and contents are added via body.innerHTML`       | PASS    |                                                |
| `Template may contain title element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain tr element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain track element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain u element. The template element and contents are added via body.innerHTML`          | PASS    |                                                |
| `Template may contain ul element. The template element and contents are added via body.innerHTML`         | PASS    |                                                |
| `Template may contain var element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |
| `Template may contain video element. The template element and contents are added via body.innerHTML`      | PASS    |                                                |
| `Template may contain wbr element. The template element and contents are added via body.innerHTML`        | PASS    |                                                |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260680357)

# Sauce (safari) #
Testing web-platform-tests at revision 73e7c426f7eb7b7c557fb1ef36e9734724a72b63
Using binary None
Starting tests

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260680383)

# Chrome (unstable) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260680388)

# Sauce (MicrosoftEdge) #

**Review by jgraham — APPROVED** [review]

File: html/semantics/scripting-1/the-template-element/template-element/template-content.html [hunk 1]
```diff
@@ -14,63 +14,49 @@
...
 });
 
-generate_tests(assert_not_equals, parameters,
-        'Template may contain any element, except the html element, '
-        + 'the head element, the body element, or the frameset element');
-
-
 
 
 var parameters = [];
```
Comment (Ms2ger) [inline html/semantics/scripting-1/the-template-element/template-element/template-content.html]: Leftover code

**Pull request merged**